src/components/GoalCard/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import Strings from '../../i18n/en';
+import ProgressBar from '../ProgressBar';
+import { GoalCardStyles } from './GoalCardStyle';
+
+export type GoalCardProps = {
```

Comment:
Can we create a file which can host all the types and from there import it into this file.

Reply:
I think we can keep it here as it is exclusive to the component.